src/v2/guide/team.md

```diff
@@ -1136,7 +1136,41 @@ order: 803
       links: [
         'https://vuejs.london'
       ]
-    }
+    },
+    {
+      name: 'Phan An',
+      title: 'Backend Designer & Process Poet',
```

Comment:
@filrak @phanan Is it a mistake that @phanan has been added here?

Reply:
Oh, indeed.